diff --git a/data/io.github.lainsce.Notejot.metainfo.xml.in b/data/io.github.lainsce.Notejot.metainfo.xml.in
index a7bbbf2..c5d12e9 100644
--- a/data/io.github.lainsce.Notejot.metainfo.xml.in
+++ b/data/io.github.lainsce.Notejot.metainfo.xml.in
@@ -17,15 +17,17 @@
     <provides>
         <binary>io.github.lainsce.Notejot</binary>
     </provides>
+    <launchable type="desktop-id">io.github.lainsce.Notejot.desktop</launchable>
     <kudos>
       <kudo>ModernToolkit</kudo>
       <kudo>HiDpiIcon</kudo>
     </kudos>
     <developer_name translatable="no">Lains</developer_name>
     <url type="homepage">https://github.com/lainsce/notejot/</url>
     <url type="bugtracker">https://github.com/lainsce/notejot/issues</url>
     <url type="donation">https://www.ko-fi.com/lainsce/</url>
     <url type="translate">https://github.com/lainsce/notejot/blob/main/po/README.md</url>
+    <url type="vcs-browser">https://github.com/lainsce/notejot/</url>
     <screenshots>
         <screenshot type="default">
             <image>https://raw.githubusercontent.com/lainsce/notejot/main/data/shot.png</image>
diff --git a/src/Views/NoteContentView.vala b/src/Views/NoteContentView.vala
index ba6c608..f895fe1 100644
--- a/src/Views/NoteContentView.vala
+++ b/src/Views/NoteContentView.vala
@@ -247,7 +247,7 @@ public class Notejot.NoteContentView : View {
             if (((Adw.Leaflet)MiscUtils.find_ancestor_of_type<Adw.Leaflet>(this)).folded) {
                 back2_button.visible = false;
             } else {
-                back2_button.visible = ((MainWindow)MiscUtils.find_ancestor_of_type<MainWindow>(this)).sgrid.get_visible_child_name () == "notegrid" != false ? true : false;
+                back2_button.visible = ((MainWindow)MiscUtils.find_ancestor_of_type<MainWindow>(this)).sgrid.get_visible_child_name() == "notegrid" ? true : false;
             }
             back2_button.clicked.connect (() => {
                 ((Adw.Leaflet)MiscUtils.find_ancestor_of_type<Adw.Leaflet>(this)).set_visible_child (((MainWindow)MiscUtils.find_ancestor_of_type<MainWindow>(this)).sgrid);
